docs/source/theory/actuatorLine.rst

```diff
@@ -0,0 +1,185 @@
+commonly used projection function is a uniform Gaussian as proposed by
+Sørensen and Shen :cite:`Sorensen:2002`,
+
+.. math:: g(\vec{r}) = \frac{1}{\pi^{3/2} \epsilon^3} e^{-\left( \left| \vec{r} \right|/\epsilon \right)^2},
+
+where :math:`\vec{r}` is the position vector between the fluid point of
+interest to a particular point on the actuator, and :math:`\epsilon` is
+the width of the Gaussian, which determines how diluted the body force
+become. As an example, for an actuator line extending from :math:`l=0`
+to :math:`L`, the body force at point :math:`(x,y,z)` due to the line is
+given by
+
+.. math::
+   :label: force-integral
+           
+   f_i(x,y,z) = \int_0^L g\left(\vec{r}\left(l\right)\right) F'_i\left(l\right) \: \textrm{d} l.
+   
+
+Here, the projection function’s position vector is a function of
+position on the actuator line. The part of the line nearest to the point in
+the fluid at :math:`(x,y,z)` has most weight.
+
+The force along an actuator line or over an actuator disk is often
+computed using blade element theory, where it is convenient to discretize
+the actuator into a set of elements. For example, with the actuator line,
+the line is broken into discrete line segments, and the force at the center
+of each element, :math:`F_i^k`, is computed. Here, :math:`k` is the actuator
+element index. These actuator points are independent of the fluid mesh.
+This set of point forces is then projected onto the fluid mesh using any
+desired projection function, :math:`g(\vec{r})`, as described above.
+This is convenient because the integral given in Equation
+:eq:`force-integral` can become the summation
+
+.. math::
+   :label: force-summation
+           
+   f_i(x,y,z) = \sum_{k=0}^N g(\vec{r}^k) F_i^k.
+   
+
+This summation well approximates the integral given in Equation
+:eq:`force-integral` so long as the ratio of actuator element size to
+projection function width :math:`\epsilon` does not exceed a certain threshold.
+
+Design
+======
+
+The initial actuator capability implemented in Nalu is focused on the actuator line algorithm. However, the class hierarchy is designed with placeholders for actuator disk and swept actuator line capability and potentially an actuator surface capability. The general set of functions required to handle actuators are the same whether it is a line, disk, swept line, or surface. The various classes implementing the different actuator capabilities inherit from an overall actuator class. The ``ActuatorLineFAST`` class couples Nalu with the third party library OpenFAST for actuator line simulations of wind turbines. OpenFAST (https://nwtc.nrel.gov/FAST), available from https://github.com/OpenFAST/openfast, is a aero-hydro-servo-elastic tool to model wind turbine developed by the National Renewable Energy Laboratory (NREL). The ``ActuatorLineFAST`` class will help Nalu effectively act as an inflow module to OpenFAST by supplying the velocity field information.
```

Comment:
Removed the placeholder classes in the latest commits. Should this be rephrased?